cmd/commands/init.go

```diff
@@ -0,0 +1,122 @@
+package commands
+
+import (
+	"io/ioutil"
+	"path"
+
+	"github.com/urfave/cli"
+
+	cmn "github.com/tendermint/go-common"
+)
+
+var InitCmd = cli.Command{
+	Name:      "init",
+	Usage:     "Initialize a basecoin blockchain",
+	ArgsUsage: "",
+	Action: func(c *cli.Context) error {
+		return cmdInit(c)
+	},
+	Flags: []cli.Flag{
+		ChainIDFlag,
+	},
+}
+
+func cmdInit(c *cli.Context) error {
```

Comment:
I like the idea of having a deterministic option strictly for tutorial usage... maybe a little bit of a discussion in the tutorial explaining this